### PR TITLE
Hit api

### PR DESCRIPTION
**Pull request by uwadonat**
### Leaderboard: hit the API

 **In this activity I implemented followings:**
-  working on the Leaderboard list app, sending and receiving data from/to the Leaderboard API.
- Make JavaScript code asynchronous

**Hope to meet all requirements of this milestone**

_Thank you for your time to review my project_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 > This project is about set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features.
 
-![Screenshot](https://github.com/uwadonat/Leaderboard/blob/setup-project/images/Screenshot.png)
+![Screenshot](https://github.com/uwadonat/Leaderboard/blob/hit-api/images/Screenshot.png)
```
Comment (Dipeshtwis) [inline README.md:9]: - [x] README `live demo` link is broken. Please fix it so that README looks professional. :slightly_smiling_face:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @uwadonat, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Submit button hit the POST API endpoint and post user entries and Refresh button hit the GET API endpoint causing the scores list to change. :muscle:


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [x] I would recommend removing the `.hintrc` file as you are no longer using `HTML` linters.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by uwadonat** [comment]
Thank you for good review.

Changes requested was implemented.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by uwadonat** [comment]
thank you for approval.

**Pull request merged**